### DISCUSSION TITLE
move minio volume to anether disk

### DISCUSSION THREAD

**Discussion by vandr0s** [minio/minio#20274] [Q&A] [answered]
Good day.
I have minio instance installed on vmware debian vm from binaries with one volume(MINIO_VOLUMES="/mnt/minio") on file system, used as gitlab container registry backend with one bucket. The size of this volume is close to 2tb. Recently i realized that partition table of disk is mbr. Will i have problems if i create another virtual disk with gpt partition table on vmware, then just stop minio, move all data to this disk with mv or rsync and create symbolic link to /mnt/minio?

**Comment by klauspost** [comment, marked as answer]
Since it is a single disk it should be ok. 

Stop minio. Copy data. Replace mount. Start minio.  No symlinks.

**Reply by vandr0s** [reply]
Thanks for quick answer! I will try in maintenance window

**Comment by vandr0s** [comment]
I have copied data to new mounted disk with rsync -av minio /mnt/data/
Minio seems working fine, thanks again!
